docs/APIs/API_Documentation.md

```diff
@@ -90,21 +90,21 @@ It requires almost no effort to translate scattered comments in code into well f
 
 ### Comment markup
 
-Documentation has special markup to describe parameters, return values, notes, code example and so on.
+Documentation has special markup to describe parameters, return values, notes, code examples and so on.
 
-Doxygen accepts reserved words prefixed with ``\`` or ``@``, where some of the commonly used are:
+Doxygen accepts reserved words prefixed with ``\`` or ``@``. Some of the commonly used ones are:
 
-* @param <name> text
+* @param <name> text.
```

Comment:
I think all of these should be in `` and without the period (because there'll be only code and it's confusing to have a period with code)
So for example * `@param <name> text`